manifests/overlays/apps/base/watson-x/watson-x-docs.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: dashboard.opendatahub.io/v1
+kind: OdhDocument
+metadata:
+  name: watson-x-use-case
+  annotations:
+    opendatahub.io/categories: 'Data preprocessing, Data visualization, Model development'
+spec:
+  appName: watson-x-ai
+  type: how-to
+  displayName: Enable your enterprise to develop and deploy machine learning models and generative AI solutions.
```

Comment:
change displayName to "Watsonx.ai use case"
description is correct

Reply:
ooph - wow.  Good catch...
![Screenshot 2024-04-04 at 1 05 28 PM](https://github.com/opendatahub-io/odh-dashboard/assets/7306953/ced3223e-1bd3-489f-817b-24e8d441cfc0)

Reply:
I notice now with the screen capture showing the two tiles side-by-side - they should be consistent
Please change the title on the How to tile from **Watsonx.ai use case** to *IBM watsonx.ai use case*

Reply:
![Screenshot 2024-04-04 at 1 19 11 PM](https://github.com/opendatahub-io/odh-dashboard/assets/7306953/af450e04-6795-4aba-86c7-36d9d75ec227)
This should be resolved now...